app/src/main/res/values/strings.xml

```diff
@@ -89,6 +89,12 @@
     <string name="pref_show_mp3_player">Show MP3 Player</string>
     <string name="pref_show_mp3_player_summary">Show/hide the MP3 player toolbar</string>
 
+    <string name="pref_audio_setting">Audio and Vibration Settings</string>
```

Comment:
I would rather call it "Rest timer settings". Audio and vibration seems too general.

Reply:
Solved by 1e17813fdd3a98904b254e5a98e8b61d2a3281bd